### PR TITLE
Module I pair programming-Create a contact me section

### PR DESCRIPTION
**Pull request by d4nQw3rty**
In this pull request, we are going to create a form section for our project.
This section will have 3 input fields.
A full name field with a max length of 30chars.
An email address accepting email format.
And a field to write a message with 500 chars in length.
All fields will be required.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -506,3 +506,104 @@ body {
...
+  /* or 118% */
+
+  display: flex;
+  align-items: center;
+  padding: 15px 36px 15px 16px;
+
+  /* Neutral/N800 */
+
+  color: #172b4d;
+}
```
Comment (Abenezer-Tilahun) [inline style.css:609]: - [ ] Good job for following the Figma design Requirement, You stick to the design very well with the font, colors, images and spacing (margin/padding) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
- [ ] Good job for implementing Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.
- [ ] For mobile screen size  the  Minimum mobile screen size should be 375px. But at 375px your contact form is not responsive , Kindly fix it. 
Your Project at 375px look like this :

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @d4nQw3rty  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README
✔️ Good commit messages

almost

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [Optional ] Kindly, try to fix the MIT LICENSE in Your README file, The Link is borken. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (d4nQw3rty) [reply style.css:609]: image
Fixed!

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @d4nQw3rty,

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**